docs/src/literate-howto/delaunay.jl

```diff
@@ -0,0 +1,242 @@
+
+# The `-1` ghost vertex refers to the outer boundary, while the `-2` ghost vertex refers to the inner boundary.
+# The ordering of these ghost vertices comes from the ordering of sections in the `boundary_nodes` variable.
+
+# ### Solving the problem
+# Solving the problem is now straightforward, and we simply mimic the implementation from the 
+# [heat equation tutorial](@ref tutorial-heat-equation). The only modifications are the use of triangles 
+# for defining the interpolation and quadrature rule, and the new boundary conditions. 
+ip = Lagrange{RefTriangle,1}()
+qr = QuadratureRule{RefTriangle}(2)
+cellvalues = CellValues(qr, ip)
+dh = DofHandler(grid)
+add!(dh, :u, ip)
+close!(dh)
+K = allocate_matrix(dh)
+
+#-
+ch = ConstraintHandler(dh)
+dbc = Dirichlet(:u, getfacetset(grid, "-1"), (x, t) -> x[1]^2 * x[2]^2)
+add!(ch, dbc)
+nbc = Dirichlet(:u, getfacetset(grid, "-2"), (x, t) -> 0.0)
+add!(ch, nbc)
+close!(ch)
+function assemble_element!(Ke::Matrix, fe::Vector, cellvalues::CellValues) # Copied exactly from the heat equation tutorial
+    n_basefuncs = getnbasefunctions(cellvalues)
+    ## Reset to 0
+    fill!(Ke, 0)
+    fill!(fe, 0)
+    ## Loop over quadrature points
+    for q_point in 1:getnquadpoints(cellvalues)
+        ## Get the quadrature weight
+        dΩ = getdetJdV(cellvalues, q_point)
+        ## Loop over test shape functions
+        for i in 1:n_basefuncs
+            δu = shape_value(cellvalues, q_point, i)
+            ∇δu = shape_gradient(cellvalues, q_point, i)
+            ## Add contribution to fe
+            fe[i] += δu * dΩ
+            ## Loop over trial shape functions
+            for j in 1:n_basefuncs
+                ∇u = shape_gradient(cellvalues, q_point, j)
+                ## Add contribution to Ke
+                Ke[i, j] += (∇δu ⋅ ∇u) * dΩ
+            end
+        end
+    end
+    return Ke, fe
+end
+function assemble_global(cellvalues::CellValues, K::SparseMatrixCSC, dh::DofHandler)
+    ## Allocate the element stiffness matrix and element force vector
+    n_basefuncs = getnbasefunctions(cellvalues)
+    Ke = zeros(n_basefuncs, n_basefuncs)
+    fe = zeros(n_basefuncs)
+    ## Allocate global force vector f
+    f = zeros(ndofs(dh))
+    ## Create an assembler
+    assembler = start_assemble(K, f)
+    ## Loop over all cels
+    for cell in CellIterator(dh)
+        ## Reinitialize cellvalues for this cell
+        reinit!(cellvalues, cell)
+        ## Compute element contribution
+        assemble_element!(Ke, fe, cellvalues)
+        ## Assemble Ke and fe into K and f
+        assemble!(assembler, celldofs(cell), Ke, fe)
+    end
+    return K, f
+end
+#md nothing # hide
+
+K, f = assemble_global(cellvalues, K, dh)
+apply!(K, f, ch)
+u = K \ f;
+
+# The solution can be visualised as in previous tutorials. Alternatively, we can use `tricontourf` which 
+# knows how to use DelaunayTriangulation.jl's `Triangulation`.
+sol =  evaluate_at_grid_nodes(dh, u, :u)
+tricontourf(tri, sol)
```

Comment:
IMHO it would be advantageous to plot with FerriteViz

Reply:
I wanted to do that originally but didn't because it doesn't work on Ferrite master. I could dev in https://github.com/Ferrite-FEM/FerriteViz.jl/pull/103 to make it work?

Reply:
thats fine in my opinion but ping @KnutAM @termi-official @fredrikekre

Reply:
Seems like a good idea to too! 
The only downside I see, is that it requires that branch to be continuously kept up to date with Ferrite#master changes now in the time before releasing. But not sure how hard / much work that is?

Reply:
I agree with Maxi, but let us do this after 1.0 release, so we can make a FerriteViz release instead of relying on me keeping the FerriteViz branch up to date.

Reply:
Keeping the branch up to date probably puts more work on you than is necessary. Another alternative is to just remove the visualisation step and simply note that it could be done, if you would rather drop the Makie usage. I just thought of `tricontourf` since it is something that also works directly with DelaunayTriangulation.jl. I don't really mind what is done here.

Reply:
Is there anything waiting on me here?

Reply:
We need a few more days on our side and will report back afterwards. Trying to wrap up the Ferrite 1.0 release, so we can update the subsequent packages and merge PRs like this one here in order. Thanks again already for your work!

Reply:
No worries at all, there's no rush. Was just checking in case you were waiting on me to do something 😅 Thanks

Reply:
I guess this is just waiting on FerriteViz to be made compatible with Ferrite 1.0 before reviewing further?

Reply:
Right. The release is already in review, so we should be able to update FerriteViz to 1.0 in the not too far future.